src/osvr/Common/PathElementTypes.cpp

```diff
@@ -89,9 +88,18 @@ namespace common {
         /// port if one isn't already specified.
         static inline std::string
         attachPortToServerIfNoneSpecified(std::string const &server, int port) {
-            auto endsWithPort = std::regex{".*:[[:digit:]]+$"};
```

Comment:
Does replacing `[[:digit:]]` with `[0-9]` work, by chance?

The underlying problem is that gcc 4.8 doesn't have full regex support.